source/includes/get-started/quickstart.c

```diff
@@ -0,0 +1,37 @@
+#include <bson/bson.h>
+#include <mongoc/mongoc.h>
+#include <stdio.h>
+
+int main(int argc, char *argv[])
```

Comment:
The standard the C team has been using appears to put spaces between function names and parentheses. See: [example](https://mongoc.org/libmongoc/current/mongoc_collection_find_with_opts.html#examples)

```suggestion
int main (int argc, char *argv[])
```

Applies to all occurrences here.